### PR TITLE
1.48 post

### PR DESCRIPTION
**Pull request by mooreds**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205866911733531
  - https://app.asana.com/0/0/1205866751488080
  - https://app.asana.com/0/0/1205866751794459

### CODE REVIEW

**Review by mark-robustelli — CHANGES_REQUESTED** [review]
Overall looks good, a couple of suggestions though.  Also, note the check is not passing for the full url.  It's probably the same thing others are experiencing.

File: astro/src/content/blog/announcing-fusionauth-1-48.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+It's not just webhook signing. There were other webhook related improvements too, including:
+
+* Webhook test messages include more information if the webhook does not succeed.
+* You can use [key master](/docs/operate/secure-and-monitor/key-master) to manage webhook certificates. This is the recommended solution going forward.
+* A bug where tenant specific webhooks were removed when `PATCH`ing a tenant has been fixed.
+* Test messages sent using the admin UI preserve the body payload, making it easier to develop webhooks.
+
+## User Search
+
+Previous to this release, there was a limit on the number of user or entity search results. You could only get 10,000 users or entities in each search response. If you had more than that, which many of our community and clients do, you had to run multiple queries and join the JSON results. While this [could be scripted](https://github.com/FusionAuth/fusionauth-example-scripts/tree/master/full-user-search), it was a hassle.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/announcing-fusionauth-1-48.mdx:54]: Hmm. It looks like only the API, but not sure if that is a bug.

File: astro/src/content/blog/announcing-fusionauth-1-48.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+publish_date: 2023-11-01
+title: Announcing FusionAuth 1.48
+description: This update includes webhook improvements, the ability to retrieve more than 10,000 users, and exposes the access token to your lambdas.
+authors: Dan Moore
+image: /img/blogs/release-1-48/fusionauth-1-48.png
```
[the remark this answers is not in the document]
Comment (mooreds) [reply astro/src/content/blog/announcing-fusionauth-1-48.mdx:6]: Added now.

**Review by mark-robustelli — APPROVED** [review]
LGTM

**Pull request merged**